frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -294,6 +294,7 @@ export const getCommonInitialValues = (
     serverless: getServerlessData(editAppResource),
     pipeline: {
       enabled: !_.isEmpty(pipelineData),
+      data: pipelineData,
```

Comment:
Also you don't need to add a new prop for storing the pipeline in the form state itself as if you go through `EditApplication` you will see that we are sending all the resources including pipeline to `createOrUpdateGitResources` so you can directly get the `pipeline` there.

Reply:
Thanks, updated.